frontend/src/components/table/TableItem.js

```diff
@@ -255,14 +255,13 @@ class TableItem extends PureComponent {
           for (let [k, v] of Object.entries(fields)) {
             editedCells[k] = v;
           }
```

Comment:
I put the setState within the if (resp[0] && resp[0].fieldsByName) { } as we want to update state only when editedCells was altered.